src/scope/server/app.py

```diff
@@ -615,40 +643,12 @@ def open_browser_when_ready(host: str, port: int, server):
         logger.info(f"🌐 UI is available at: {url}")
 
 
-def main():
-    """Main entry point for the daydream-scope command."""
-    parser = argparse.ArgumentParser(
-        description="Daydream Scope - Real-time AI video generation and streaming"
-    )
-    parser.add_argument(
-        "--version",
-        action="store_true",
-        help="Show version information and exit",
-    )
-    parser.add_argument(
-        "--reload",
-        action="store_true",
-        help="Enable auto-reload for development (default: False)",
-    )
-    parser.add_argument(
-        "--host", default="0.0.0.0", help="Host to bind to (default: 0.0.0.0)"
-    )
-    parser.add_argument(
-        "--port", type=int, default=8000, help="Port to bind to (default: 8000)"
-    )
-    parser.add_argument(
-        "-N",
-        "--no-browser",
-        action="store_true",
-        help="Do not automatically open a browser window after the server starts",
-    )
-
-    args = parser.parse_args()
+def run_server(reload: bool, host: str, port: int, no_browser: bool):
+    """Run the Daydream Scope server."""
 
-    # Handle version flag
-    if args.version:
-        print_version_info()
-        sys.exit(0)
+    from scope.core.pipelines.registry import (
+        PipelineRegistry,  # noqa: F401 - imported for side effects (registry initialization)
```

Comment:
The PipelineRegistry registers pipelines at import time right now so this is just to avoid unused import errors/warnings while still ensuring that the pipeline registration happens on server startup.